tools/isobuild/app-test-files.js

```diff
@@ -8,6 +8,18 @@ export const TEST_FILENAME_REGEXPS = [
   /^tests?./
```

Comment:
At this point we might consider just getting rid of the `tests?.*` form... Not only does it stick out in it's difference from the unit test and integration test filenames, and makes it harder to exactly explain how this feature works -- it also makes colocating tests not as great. It's much better to see `foo.js` and `foo.test.js` right next to each other in your editor than a bunch of files and then a bunch of `test_*` files where you can't even easily see which files have tests.

What do you think?